### PR TITLE
Vet Clinic Database: 3rd delivery "Query Multiple Tables"

### PR DESCRIPTION
**Pull request by lmunozm1702**
### Write queries (using JOIN) to answer the following questions:

image

image

image

image

image

image

image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -145,7 +145,83 @@ VALUES
...
+/* Insert data into species table */
+INSERT INTO
+    species (name)
+VALUES
+    ('Pokemon'),
+    ('Digimon');
+
+/* Modify inserted animals so it includes the species_id value */
+/* SELECT * FROM animals
+WHERE name LIKE '%mon'; */
```
Comment (TadesseDev) [inline data.sql]: - [x] Dear @lmunozm1702, Your code is clean and to keep it that way lets not leave commented code on the codebase, remember if you ever wanna see the code you have wrote once you can use your commit history to go back.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @lmunozm1702 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highligt
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (lmunozm1702) [reply data.sql]: Hi @Tadesse-Alemayehu it's done!.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @lmunozm1702 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks.
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (TadesseDev) [reply data.sql]: Good job @lmunozm1702

**Pull request merged**